test/e2e/node_feature_discovery.go

```diff
@@ -84,11 +106,22 @@ func cleanupNode(cs clientset.Interface) {
 				}
 			}
 
+			// Remove taints
+			for _, taint := range node.Spec.Taints {
+				if strings.HasPrefix(taint.Key, nfdv1alpha1.AnnotationNs) {
```

Comment:
Using `nfdv1alpha1.AnnotationNs` here looks a bit strange. Maybe you could e.g. define a const here like `TestTaintNs` or smth?

Reply:
Hmm why do you think so? All the taints that we are setting have prefix of `nfdv1alpha1.AnnotationNs` (i.e. `nfd.node.kubernetes.io`).

Reply:
My point was to specify a separate const for the test taints as they have in principle nothing to do with annotations even if the string value is the same

Reply:
we are using the same NS for the taints annotation too https://github.com/kubernetes-sigs/node-feature-discovery/blob/8a153c12cf77b06d2f8c80a31d66a90d0a7fc58e/pkg/apis/nfd/v1alpha1/annotations_labels.go#L48. But fine with me to replace it with a const.

Reply:
Done.

Reply:
> we are using the same NS for the taints annotation too

Yes we are, but that's an annotation, not a taint. I don't know if I'm thinking it in too complicated way 😅 But I feel that as they're independent/separate, better to specify a separate const

Reply:
Sure, you have a point that they are separate things.